hapi/hedera-protobufs/services/custom_fees.proto

```diff
@@ -173,3 +173,59 @@ message AssessedCustomFee {
    */
   repeated AccountID effective_payer_account_id = 4;
 }
+
+/**
+ * A custom fee definition for a consensus topic.
+ *
+ * This fee definition is specific to an Hedera Consensus Service (HCS) topic
+ * and SHOULD NOT be used in any other context.<br/>
+ * All fields for this message are REQUIRED.<br/>
+ * Only "fixed" fee definitions are supported because there is no basis for
+ * a fractional fee on a consensus submit transaction.
+ */
+message ConsensusCustomFee {
+  /**
+   * A fixed custom fee.
+   * <p>
+   * The amount of HBAR or other token described by this `FixedFee` SHALL
+   * be charged to the transction payer for each message submitted to a
+   * topic that assigns this consensus custom fee.
+   */
+  FixedFee fixed_fee = 1;
+
+  /**
+   * A collection account identifier.
+   * <p>
+   * All amounts collected for this consensus custom fee SHALL be transferred
+   * to the account identified by this field.
+   */
+  AccountID fee_collector_account_id = 2;
+}
+
+/**
+ * A wrapper for consensus custom fee list.
+ * <p>
```

Comment:
For `message` specification we do not need the paragraph tags.
The draft [style guidelines](https://github.com/hashgraph/hedera-protobufs/blob/6206557e0fdaf283a99bd0e247e74920083b7b79/documents/Specification-Format-Style-Guidelines.md) may help here.

Reply:
Fixed.
Thanks for pointing out the guidelines, it's very helpful.